src/mobility/components/OperatorActionButton.tsx

```diff
@@ -61,8 +61,16 @@ export const OperatorActionButton = ({
     await Linking.openURL(url).catch(() =>
       showAppMissingAlert({appStoreUri, operatorName}),
     );
-    // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [operatorName, benefit, rentalAppUri, appStoreUri, valueCode]);
+  }, [
+    analytics,
+    operatorName,
+    benefit,
+    isUserEligibleForBenefit,
+    rentalAppUri,
+    valueCode,
+    showAppMissingAlert,
+    appStoreUri,
+  ]);
```

Comment:
Is this `handleCallToAction` just an event handler? In that case is doesn't need `useCallback` at all.